src/main/java/com/jherrick/BigBoard.java

```diff
@@ -265,31 +234,164 @@ public boolean isDrawn() {
 		// Yeah... so... this needs to be built!
 		return false;
 	}
-	
-	public int evaluate() {
-		if (state == BoardState.X_WON) {
-			return Integer.MAX_VALUE;
-		}
-		if (state == BoardState.O_WON) {
-			return Integer.MIN_VALUE;
-		}
-		int score = 0;
-		for (int i = 0; i < boards.length; i++) {
-			// score += Eval.getValues().get(boards[i]);
-		}
-		return 10 * c0 * Eval.twoInARowsWithOpenThird(xWinBoards, (oWinBoards | drawnBoards))
-				+ 10 * c1 * Eval.middleSquare(xWinBoards)
-				- 10 * c0 * Eval.twoInARowsWithOpenThird(oWinBoards, (xWinBoards | drawnBoards))
-				- 10 * c1 * Eval.middleSquare(oWinBoards) + score;
-	}
 
 	/**
 	 * @override toString() method
 	 */
 	public String toString() {
 		StringBuilder result = new StringBuilder("\n  HCN: " + Utils.boardToHCN(this) + "\n");
+		
+		result = subBoardPrint(result);
+
+		String bottomKey = String.format("%3s a  b  c %1s d  e  f %1s g  h  i \n", " ", " ", " ");
+		result.append(bottomKey);
+
+		return result.toString();
+	}
+
+
+
+
+	// ===================== Helper Functions ========================
+
+	/**
+	 * Toggles which side is to move
+	 **/
+	private void toggleSide() {
+		side = side == Side.X ? Side.O : Side.X;
+	}
+
+
+	// =========== HCN BigBoard Constructor Helper Functions =========
+
+	private boolean isValidHCN(String hcn){
```

Comment:
We should definitely create a class of HCN Utils. Not necessary for this PR though.